src/spdx2/agent/spdx2.php

```diff
@@ -436,8 +436,12 @@ protected function addScannerResults(&$filesWithLicenses, ItemTreeBounds $itemTr
     {
       $reportedLicenseId = $this->licenseMap->getProjectedId($row['rf_fk']);
       $shortName = $this->licenseMap->getProjectedShortname($reportedLicenseId);
-      if ($shortName != 'No_license_found' && $shortName != 'Void') {
-        $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = $shortName;
+      if ($shortName != 'Void') {
+        if($shortName != 'No_license_found'){
+          $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = $shortName;
+        }else{
+          $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = "";
```

Comment:
@mcjaeger @kestewart : is it correct to state `LicenseInfoInFile: NONE` if the scanners were not able to find any license? or should we act defensively and say `NOASSERTION`?